### DISCUSSION TITLE
挂载一刻相册后打开图片出现failed link: failed get link: not found 302 redirect

### DISCUSSION THREAD

**Discussion by bittysense** [AlistGo/alist#7590] [General]
![IMG_0191](https://github.com/user-attachments/assets/ec0c0a76-23ee-41d4-b0f4-adddd758e4f7)
![IMG_0192](https://github.com/user-attachments/assets/8eeedd3d-691b-4668-82e2-71c39940fb36)

**Comment by zhujitop** [comment]
兄弟 解决没？
我的也这样，换成别的就无法加载。

**Reply by bittysense** [reply]
> 兄弟 解决没？ 我的也这样，换成别的就无法加载。

不用这一刻相册了，辣鸡
